content/getting-started/yaml.md

```diff
@@ -307,3 +302,69 @@ scripts:
     echo 'This is a Post-publish script'
     echo 'This script is multiline'
 ```
+
+## Conditional execution
+### Skip building a commit
+
+If you do not wish Codemagic to build a particular commit, include `[skip ci]` or `[ci skip]` in your commit message.
+
+### Watch for changes in files
+
+You can avoid unnecessary builds when functional components of your repository were not modified. Use conditional workflow execution to skip building workflow if watched files were not updated since the last successful build.
+
+You should specify files to watch in `changeset` by using `includes` and `excludes` keys.
+
+```yaml
+workflows:
+  build-app:
+    name: Build App
+    triggering:
+      events:
+        - push
+    when:
+      changeset:
+        includes:
+          - '.'
+        excludes:
+          - '**/*.md'
+```
+
+In this case, build would be skipped if there were no changes to the repository except Markdown files `.md`.
+
+Note that `codemagic.yaml` is always included in change set by default.
+
+Both keys `includes` and `excludes` in `changeset` are *optional*. If `includes` key is not specified, its value would defalut to `'.'`. `excludes` value defaults to an empty array.
```

Comment:
empty array is not a good docs terminology.
excludes defaults to nothing? or no exclusions?